responsibleai/requirements.txt

```diff
@@ -1,7 +1,7 @@
 dice-ml>=0.7.2,<0.8
 econml~=0.12.0
 jsonschema
-erroranalysis>=0.1.27
+erroranalysis>=0.1.28
```

Comment:
should we also have an upper bound for all packages?
At least to next major version?

Reply:
erroranalysis has always been backwards compatible, so no upper bound has been needed so far